### PR TITLE
Add tags from filter_categories

### PR DESCRIPTION
**Pull request by drewbutlerbb4**
Adds the tags from asset.filter_categories into the Featured Page cards.
Signed-off-by: Andrew-Butler <Andrew.Butler@ibm.com>

Screen Shot 2021-09-09 at 9 32 13 AM

### CODE REVIEW

**Comment by drewbutlerbb4** [comment]
@ckadner the default for tags is to not have any tags if you think there should be a default tag I can add one.

File: dashboard/origin-mlx/src/pages/MetaFeaturedPage/MetaCard.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ function MetaCard(props: MetaCardProps) {
 
   const description = firstSentence(props.description || 'Component for your Pipelines.')
 
-  const tags = tag.includes(',') ? tag.split(',') : [tag]
+  const tags = Object.keys(asset.filter_categories).map((key: any) => asset.filter_categories[key])
```
Comment (ckadner) [inline dashboard/origin-mlx/src/pages/MetaFeaturedPage/MetaCard.tsx:83]: does this work if the object has no `filter_categories`? i.e. pipelines don't have it

**Comment by ckadner** [comment]
> @ckadner the default for tags is to not have any tags if you think there should be a default tag I can add one.

no, no default tags

File: dashboard/origin-mlx/src/pages/MetaFeaturedPage/MetaCard.tsx (hunk 1 above)
Comment (drewbutlerbb4) [reply dashboard/origin-mlx/src/pages/MetaFeaturedPage/MetaCard.tsx:83]: Ah I thought we made the filter_categories tag mandatory for all assets, but I forgot that pipelines are different. I'll fix that.

**Pull request closed without merging**